tests/mol-rendering-audio/EPUB/package.opf

```diff
@@ -4,33 +4,31 @@
     <dc:coverage>Media Overlays</dc:coverage>
     <dc:creator>Marisa DeMeglio</dc:creator>
     <dc:date>2022-02-10</dc:date>
-    <dc:description>Tests whether the Reading System implements Media Overlays' RS Spec "Rendering Audio" section.</dc:description>
+    <dc:description>
+      The reading system must respect the requirements for clipBegin and clipEnd when playing audio.
+    </dc:description>
     <dc:identifier id="pub-id">mol-rendering-audio</dc:identifier>
     <dc:language>en</dc:language>
-    <dc:title>Media Overlays Tests: Rendering Audio</dc:title>
+    <dc:title>Media overlays render audio</dc:title>
     <link rel="dcterms:rights" href="https://www.w3.org/Consortium/Legal/2015/copyright-software-and-document"/>
     <link rel="dcterms:rightsHolder" href="https://www.w3.org"/>
-    <meta property="belongs-to-collection">should</meta>
```

Comment:
Same: I removed this entirely, because https://www.w3.org/TR/epub-rs-33/#sec-rsconf-rendering-audio appears to be a MUST, not a SHOULD (although it has one SHOULD at the end, which I think is untested). Please let me know if I misunderstood something.

Reply:
Same answer...